tests/ops/test_channel_ops.py

```diff
@@ -279,6 +279,40 @@ def test_p0_p1_sum_not_normalized(self):
         with pytest.raises(ValueError, match="must be between"):
             channel.ResetError(1.0, 1.0, wires=0).kraus_matrices
 
+    @pytest.mark.parametrize("angle", np.linspace(0, 2 * np.pi, 7))
+    def test_grad_reseterror(self, angle, tol):
+        """Test that analytical gradient is computed correctly for different states. Channel
```

Comment:
Maybe worth adjusting the docstring here, as we're falling back to finite differences. `qml.ResetError` doesn't support analytic gradients at the moment, [`grad_method = "F"` here denotes this fact](https://github.com/PennyLaneAI/pennylane/blob/62f7c10766816322862d77d330ba83274fb1ff59/pennylane/ops/channel.py#L345).

Reply:
I have removed the word `analytical`. I hope it looks good now